docs/how-to/how-to-pull-images-in-guest-with-kata.md

```diff
@@ -178,135 +178,217 @@ Currently, the image pulled in the guest will be downloaded and unpacked in the
+   ```
+
+6. Deploy
+
+    ```bash
+    kubectl apply -f local-storage-class.yaml
+    ```
+
+7. Create `trusted-storage-pv.yaml` file to define Persistent Volume (PV) for the block device
+   ```yaml
+   ---
+   apiVersion: v1
+   kind: PersistentVolume
+   metadata:
+     name: trusted-block-pv
+   spec:
+     capacity:
+       storage: ${STORAGE_SIZE}
+     volumeMode: Block
+     accessModes:
+       - ReadWriteOnce
+     persistentVolumeReclaimPolicy: Retain
+     storageClassName: local-storage
+     local:
+       path: ${LOOP_DEVICE}
+     nodeAffinity:
+       required:
+         nodeSelectorTerms:
+           - matchExpressions:
+               - key: kubernetes.io/hostname
+                 operator: In
+                 values:
+                   - ${NODE_NAME}
+   ```
+
+8. Deploy:
+
+   ```bash
+   envsubst < trusted-storage-pv.yaml | kubectl apply -f -
+   ```
+
+9. Create `trusted-storage-pvc.yaml` file to define Persistent Volume Claim (PVC) for the block device
+   ```yaml
+   ---
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: trusted-pvc
+   spec:
+     accessModes:
+       - ReadWriteOnce
+     resources:
+       requests:
+         storage: ${STORAGE_SIZE}
+     volumeMode: Block
+     storageClassName: local-storage
+   ```
+
+10. Deploy:
+
+    ```bash
+    envsubst < trusted-storage-pvc.yaml | kubectl apply -f -
+    ```
+
+11. Create a file `trusted-storage-pod.yaml` with pulling large image in guest
+
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: large-image-pod
+    spec:
+      runtimeClassName: kata-qemu
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/hostname
+                    operator: In
+                    values:
+                      - ${NODE_NAME}
+      volumes:
+        - name: trusted-storage
+          persistentVolumeClaim:
+            claimName: trusted-pvc
+      containers:
+        - name: app-container
+          image: quay.io/confidential-containers/test-images:largeimage
+          command: ["/bin/sh", "-c"]
+          args:
+            - sleep 6000
+          volumeDevices:
+            - devicePath: /dev/trusted_store
+              name: trusted-storage
+    ```
+
+12. Deploy:
+
+    ```bash
+    envsubst < trusted-storage-pod.yaml | kubectl apply -f -
+    ```
+
+13. When you enter the pod, you can see that the additional storage is present:
+
+    ```bash
+    df -h
+    ```
+
+    See `overlay` of size `10G` mounted on `/`:
+    ```text
+    Filesystem                Size      Used Available Use% Mounted on
+    overlay                   9.7G      2.0G      7.2G  22% /
+    tmpfs                    64.0M         0     64.0M   0% /dev
+    tmpfs                   927.8M         0    927.8M   0% /sys/fs/cgroup
+    (...)
+    ```
```

Comment:
```suggestion
```bash
$ df -h
  Filesystem                Size      Used Available Use% Mounted on
  overlay                   9.7G      2.0G      7.2G  22% /
  tmpfs                    64.0M         0     64.0M   0% /dev
  tmpfs                   927.8M         0    927.8M   0% /sys/fs/cgroup
  (...)
```
See `overlay` of size `10G` mounted on `/`
```

Reply:
@Apokleos I did the split of command and output on purpose. I find it hard to follow all commands that are to be invoked because I need to copy each line manually from the code block. It is easier to copy the command directly using the `Copy` button and compare it to output in the next block.